web-portal/components/AddNewVenue.vue

```diff
@@ -85,9 +104,46 @@
     computed: {
       venueRequiredFields: function () {
         return this.new_venue.name !== '' &&
-          this.new_venue.address !== '' &&
+          this.new_venue.street !== '' &&
           this.new_venue.city !== '' &&
+          this.new_venue.state !== '' &&
           this.new_venue.zip !== ''
+      },
+      suggestedCities: function () {
+        return [
+          'Lexington',
+          'Danville',
+          'Versailles',
+          'Frankfort',
+          'Berea',
+          'Richmond',
+          'Whitesburg',
+          'Harrodsburg',
+          'Georgetown',
+          'Louisville',
+          'Midway',
+          'Winchester',
+          'Nicholasville',
+          'Cincinnati',
+          'Covington',
+          'Morehead',
+          'Pikeville',
+          'Hazard',
+          'Paris',
+          'Mount Sterling'
+        ]
+      },
+      suggestedStates: function () {
+        return [
+          'Kentucky',
+          'Illinois',
+          'Indiana',
+          'Missouri',
+          'Ohio',
+          'Tennessee',
+          'Virginia',
+          'West Virginia'
+        ]
```

Comment:
Thinking about this a little more and I'm wondering if there needs to be a comment for each of these lists to remind us in a year or two of why we chose the ones we did, in the order we did.

Reply:
Or, alternatively, for the benefit of hypothetical future project forkers who won't have the context of today's discussion (although the issue does summarize the reasoning, so maybe I'm overthinking it).

Reply:
i like that idea!

regarding the order of states, i just put them in alphabetical order with Kentucky at the top, but am open to other suggestions. if we took the same approach as we did with cities, we'd narrow the list to KY, OH (Cincinnati & Lorain), and WV (Parkersburg) (i _think_ that list is right based on eyeballing the list of cities, but we can reality-check the list of states with Jason's cleaned-up location data).